SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -837,7 +838,7 @@ void ASpatialFunctionalTest::AddStepSetTagDelegation(FName Tag, int32 ServerWork
 	{
 		return;
 	}
-	if (ServerWorkerId >= GetNumExpectedServers())
+	if (ServerWorkerId > GetNumExpectedServers())
```

Comment:
ServerWorkerId is 1 indexed this is the issue right? Worth mentioning in the comment above perhaps?